proposals/csharp-9.0/records.md

```diff
@@ -197,6 +197,115 @@ If the containing record is abstract, the synthesized clone method is also abstr
 If the "clone" method is not abstract, it returns the result of a call to a copy constructor. 
 
 
+### Printing members: print and ToString methods
+
+If the record is derived from `object`, the record includes a synthesized "print" method equivalent to a method declared as follows, but with a compiler-reserved name:
+```C#
+void print(System.StringBuilder builder, bool includeSeparator);
+```
+The property is `private` if the record type is `sealed`. Otherwise, the property is `virtual` and `protected`.
+
+The method:
+1. appends a separator ", " to `builder` if `includeSeparator` is true and the record has public fields or properties,
+2. for each of the record's public field and property members, appends that member's name followed by " = " followed by the member's value: `this.member`, separated with ", ",
+
+If the record type is derived from a base record `Base`, the record includes a synthesized override equivalent to a "print" method declared as follows:
+```C#
+protected override void print(StringBuilder builder, bool includeSeparator);
+```
+
+The method appends the same contents to `builder` as described above, but also then calls the base "print" method with two arguments:
+1. its `builder` parameter,
+2. a true `includeSeparator` argument if either (a) the record has public fields or properties, or (b) its `includeSeparator` parameter was given as true.
+
+The record includes a synthesized method equivalent to a method declared as follows:
+```C#
+public override string ToString();
+```
```

Comment:
Would it make sense to skip this for `abstract record`s? Or should this method be kept even for `abstract record`s, to prepare for the future where classes can inherit from records?